### PR TITLE
unpacker for uploaded archives

### PR DESCRIPTION
**Pull request by MaximilianWagner**
Implement the unpack functionality to provide emba the bare images.
Integrated the unpack in the upload so there are only unzipped images stored.
Added 2 testcases.

### CODE REVIEW

File: embark/manage.py [hunk 1]
```diff
@@ -2,11 +2,23 @@
 """Django's command-line utility for administrative tasks."""
 import os
 import sys
+import logging
 
 
 def main():
     """Run administrative tasks."""
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'embark.settings')
+
```
Comment (diegiesskanne) [inline embark/manage.py:11]: you could explain what the code is doing in the comments

File: embark/uploader/test_unpacker.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+from .unpacker import unpacker
+import unittest as ut
+
+
+class unpacker_test(TestCase):
+
+    def setUp(self):
+        self.unpacker = unpacker
+
+    def test_supported_formats(self):
```
Comment (diegiesskanne) [inline embark/uploader/test_unpacker.py:15]: don't forget to comment:)

**Review by diegiesskanne — APPROVED** [review]
clean coding keep up the good work

File: embark/manage.py (hunk 1 above)
Comment (MaximilianWagner) [reply embark/manage.py:11]: beautifying the log message following the scheme:
DATE - TIME - USER - LOG LEVEL - Message

File: embark/uploader/test_unpacker.py (hunk 2 above)
Comment (MaximilianWagner) [reply embark/uploader/test_unpacker.py:15]: Will do next time.  :)
Tried to use mnemonic names.

**Pull request merged**